core/jvm/src/test/scala/fs2/MemorySanityChecks.scala

```diff
@@ -11,14 +11,14 @@ object ResourceTrackerSanityTest extends App {
 }
 
 object RepeatPullSanityTest extends App {
-  def id[A]: Pipe[Pure, A, A] = _ repeatPull Pull.receive1 { case h #: t => Pull.output1(h) as t }
+  def id[A]: Pipe[Pure, A, A] = _ repeatPull Pull.receive1 { case (h, t) => Pull.output1(h) as t }
```

Comment:
just for curiosity. If we make receive1 to take Function2, then I think that case is redundant?